docs/404.md

```diff
@@ -2,4 +2,75 @@
+| [https://dimagi.github.io/commcare-cloud/setup/manual\_installation](https://dimagi.github.io/commcare-cloud/setup/manual_installation)                                                         | [https://commcare-cloud.readthedocs.io/en/latest/installation/2-manual-install.html](https://commcare-cloud.readthedocs.io/en/latest/installation/2-manual-install.html)                                                                                                         |
+| [https://dimagi.github.io/commcare-cloud/setup/installation](https://dimagi.github.io/commcare-cloud/setup/installation)                                                                        | [https://commcare-cloud.readthedocs.io/en/latest/reference/1-commcare-cloud/1-installation.html#](https://commcare-cloud.readthedocs.io/en/latest/reference/1-commcare-cloud/1-installation.html#)                                                                               |
+| [https://dimagi.github.io/commcare-cloud/setup/new\_environment](https://dimagi.github.io/commcare-cloud/setup/new_environment)                                                                 | [https://commcare-cloud.readthedocs.io/en/latest/installation/index.html](https://commcare-cloud.readthedocs.io/en/latest/installation/index.html)                                                                                                                               |
+| [https://dimagi.github.io/commcare-cloud/firefighting/celery](https://dimagi.github.io/commcare-cloud/firefighting/celery)                                                                      | [https://commcare-cloud.readthedocs.io/en/latest/reference/firefighting/celery.html](https://commcare-cloud.readthedocs.io/en/latest/reference/firefighting/celery.html)                                                                                                         |
+|                                                                                                                                                                                                 |                                                                                                                                                                                                                                                                                  |
+| [https://dimagi.github.io/commcare-cloud/firefighting/index](https://dimagi.github.io/commcare-cloud/firefighting/index)                                                                        | [https://commcare-cloud.readthedocs.io/en/latest/reference/firefighting/index.html](https://commcare-cloud.readthedocs.io/en/latest/reference/firefighting/index.html)                                                                                                           |
+                                        |
```

Comment:
Is this row malformed? When I view the rendered version of this file the rows below here aren't in a table format.

Reply:
I have just fixed it.